accessibility/common-remediations.md

```diff
@@ -793,6 +794,29 @@ We _don't_ do this:
   <h3>Headline Topic 2</h3>
 ```
 
+### Missing heading
+
+#### What's the problem?
+
+A section or context lacks a heading. It may be a section on a page, or a standalone context like a modal dialog. 
+
+#### Who's affected by the problem?
+
+* Screen reader users
+* Users with cognitive disabilites or challenges
+
+#### Why's it a problem?
+
+Headings allow users to understand the structure of a page. When new sections or contexts are introduced without them, it can be difficult for users to understand how the content relates to the rest of the page. 
+
+If a screen reader user is taken to a new context (e.g. a modal dialog) without a heading, it can be difficult for them to understand what's just happened without tabbing through the rest of the elements to work it out.
+
+#### How do I fix it? 
+
+When you change context or open a new section, use a heading to group the content. 
```

Comment:
```suggestion
When you change context or open a new section, use a heading to signpost the content. 
```
Unsure about this suggestion. I'm not sure headings "group" content exactly, but I am also not sure how understandable the signpost metaphor is. Enough, maybe?

Reply:
Yeah, I wasn't happy with "group" either as it feels a bit weak as a descriptor. "Signpost" is better but falls foul of the "avoid metaphors" rule, as does "telegraph" or "flag" 😩 

"Announce"? "Signify"? 

Or maybe "open" or "begin the new content"? 🤔

Reply:
"Announce" I like, "Signify" (and "demark") more literal but not particularly plain language.

"begin the new content" yep also like. "open", you've just talked about opening a new section, so duplication.

"label" i quite like but would confuse with the HTML element.

I think this is tricky because we're looking for a dual-purpose phrase that needs to relate to "providing a landmark" as this is about navigation, but it's also about describing the content. "Announce" comes close?

I also prefer "begin a new section" I think, rather than "open" in this context, as you have been discussing modals.

"When you change context or begin a new section, use a heading to announce the content." ?